### PR TITLE
sigopt-examples project in experiment create

### PR DESCRIPTION
**Pull request by taylorjacklespriggs**
@benhsu75

### CODE REVIEW

File: examples/parallel.py [hunk 1]
```diff
@@ -46,6 +46,7 @@ def evaluate_metric(self, assignments):
   conn = Connection(client_token=client_token)
   experiment = conn.experiments().create(
     name="Parallel Test Franke Function",
+    project="sigopt-examples",
```
Comment (alexandraj777) [inline examples/parallel.py]: typo?

Comment (alexandraj777) [reply examples/parallel.py]: Or, since the two look different, is one a typo and one is not?

Comment (taylorjacklespriggs) [reply examples/parallel.py]: yeah missing the `s` in the previous example

**Comment by benhsu75** [comment]
lgtm

**Pull request merged**